packages/amplify-provider-awscloudformation/src/__tests__/initializer.test.ts

```diff
@@ -10,6 +10,9 @@ jest.mock('../aws-utils/aws-cfn');
 jest.mock('fs-extra');
 jest.mock('../amplify-service-manager');
 jest.mock('amplify-cli-core');
+jest.mock('../permission-boundary/permission-boundary', () => ({
```

Comment:
Why a mocking a specific function on this file. Doesn't just mocking the file work. The problem with returning a object with specific function causes problem when new functions are added and they have to be mocked as well
```suggestion
jest.mock('../permission-boundary/permission-boundary');
```